test/runtests.jl

```diff
@@ -20,36 +20,48 @@ end
 ### Disable logging output if true (default)
 hide_logs = Pkg.get_bool_env("JULIA_PKG_TEST_QUIET", default="true")
 
-### Send all Pkg output to a BufferStream if true (default)
-hide_stdoutstderr = hide_logs
+logdir = get(ENV, "JULIA_TEST_VERBOSE_LOGS_DIR", nothing)
+### Send all Pkg output to a file called Pkg.log
+
+islogging = logdir !== nothing
+
+if islogging
+    logfile = joinpath(logdir, "Pkg.log")
+    Pkg.DEFAULT_IO[] = open(logfile, "a")
+elseif hide_logs
+    Pkg.DEFAULT_IO[] = Base.BufferStream()
+else
+    Pkg.DEFAULT_IO[] = stdout
+end
 
-Pkg.DEFAULT_IO[] = hide_stdoutstderr ? Base.BufferStream() : stdout
 Pkg.REPLMode.minirepl[] = Pkg.REPLMode.MiniREPL() # re-set this given DEFAULT_IO has changed
 
 include("utils.jl")
 
 Logging.with_logger(hide_logs ? Logging.NullLogger() : Logging.current_logger()) do
     @testset "Pkg" begin
-        @testset "$f" for f in [
-            "new.jl",
-            "pkg.jl",
-            "repl.jl",
-            "api.jl",
-            "registry.jl",
-            "subdir.jl",
-            "artifacts.jl",
-            "binaryplatforms.jl",
-            "platformengines.jl",
-            "sandbox.jl",
-            "resolve.jl",
-            "misc.jl",
-            "force_latest_compatible_version.jl",
-            "manifests.jl",
-            ]
-            @info "==== Testing `test/$f`"
-            flush(Pkg.DEFAULT_IO[])
-            include(f)
-        end
+                @testset "$f" for f in [
+                    "new.jl",
+                    "pkg.jl",
+                    "repl.jl",
+                    "api.jl",
+                    "registry.jl",
+                    "subdir.jl",
+                    "artifacts.jl",
+                    "binaryplatforms.jl",
+                    "platformengines.jl",
+                    "sandbox.jl",
+                    "resolve.jl",
+                    "misc.jl",
+                    "force_latest_compatible_version.jl",
+                    "manifests.jl",
+                    ]
+                    @info "==== Testing `test/$f`"
+                    flush(Pkg.DEFAULT_IO[])
+                    include(f)
+                end
```

Comment:
Why the extra indent?

Reply:
It's how it was before, Not exactly sure why :)